common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.mediaservice.lib.collections
+
+import com.gu.mediaservice.model.Collection
+
+object CollectionsManager {
+  def onlyLatest(collections: List[Collection]): List[Collection] = {
```

Comment:
So basically feeling if it was an audit trail, it would be more useful on the collection (of collections) rather than the singular item.